src/MerchantCenter/MerchantStatuses.php

```diff
@@ -799,7 +799,7 @@ protected function get_valid_issue_types(): array {
 	 */
 	protected function parse_presync_issue_text( string $text ): array {
 		$matches = [];
-		preg_match( '/^\[([^\]]+)\]\s*(.+)$/', $text, $matches );
+		preg_match( '/^\[([^\]]+\]?)\]\s*(.+)$/', $text, $matches );
```

Comment:
Also would be nice to add what text is inside the PHPDOC `$text` param

Reply:
This comes from the fact that we map the errors to a string `[attributeName] Error message` as seen here: https://github.com/woocommerce/google-listings-and-ads/blob/2.3.3/src/Google/BatchInvalidProductEntry.php#L84
They then get saved as a collection of errors in product meta data.

However when we display the product issues table we decided later on to include errors from multiple sources. To make them match other errors the format needed to be the other way around. So this function is basically converting:
`[attributeName] Error message` > `Error message [attributeName]`

What my fix does is catch the extra bracket since the attributeName can be `[name[0]]` in the case of an image array. I'll go ahead and add some more info in the doc block to make that clearer though.